### DISCUSSION TITLE
Wrong default sortBy behaviour - uppercased strings go first

### DISCUSSION THREAD

**Discussion by MikeyZat** [TanStack/table#2720] [Ideas]
**Describe the bug (required)**
While using the default sorting method, the **uppercased** string goes before **lowercased**. People usually 

**Provide an example via Codesandbox! (required)**
Check [codesandbox](https://codesandbox.io/s/old-haze-q8vnr?file=/src/App.js) to reproduce the issue.
![image](https://user-images.githubusercontent.com/41756225/92735995-275a2400-f37a-11ea-9b72-49b2e4b2dfee.png)

**Steps To Reproduce (required)**

1. Click on sort arrow on **status** column
2. See that uppercased examples goes before lowercased

**Expected behavior (Recommended)**
People usually uses javascript `.localeCompare()` method to perform comparision between different strings. I believe that adding this option to sorthMethods would be appreciated.

**Screenshots**

**Desktop (please complete the following information):**

- Windows
- Browser - all browsers, js logical issue

**Additional context**
I would like to create a PR with the solution

**Comment by gargroh** [comment]
U can provide your own custom sort method for the same.

**Comment by MikeyZat** [comment]
Sure but I thought that it would be nice add this option to sortTypes.js in order to support sorting uppercased & lowercased strings out of the box.

**Comment by charliwize** [comment]
I have the same problem with react sorting, is there a default way that `react-table` handles this, or is it just by providing a custom sort method?

**Comment by junagao** [comment]
Hi, I'm having the same issue. How to implemented custom sort to solve it?
